docs/source/user/jitclass.rst

```diff
@@ -40,11 +40,49 @@ initializing each defined fields.  Uninitialized fields contains garbage data.
 Methods and properties (getters and setters only) can be defined.  They will be
 automatically compiled.
 
-Specifying ``numba.typed`` containers as class members
-======================================================
-It is often desirable to use a ``numba.typed.Dict`` or a ``numba.typed.List`` as
-a class member in a ``jitclass``. Methods for using these types and various
-common patterns are presented in the following:
+
+Inferred class member types with ``as_numba_type``
```

Comment:
The heading should also mention "type annotations", which is a more familiar term to look for.

Reply:
Done.